<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>6x12 SoundPad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="CMakeLists.qrc">
    <normaloff>:/content/6x12 label.png</normaloff>:/content/6x12 label.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>801</width>
     <height>501</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>799</width>
      <height>499</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>290</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>290</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>799</width>
       <height>499</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>290</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>12</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>11</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>290</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>C059 [UKWN]</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>pushButton_1</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>pushButton_9</zorder>
    <zorder>pushButton_8</zorder>
    <zorder>pushButton_12</zorder>
    <zorder>pushButton_11</zorder>
    <zorder>pushButton_7</zorder>
    <zorder>pushButton_10</zorder>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bind sound</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Num</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>System-ui</family>
     <pointsize>26</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 26pt &quot;System-ui&quot;;</string>
   </property>
   <property name="text">
    <string>6x12 SoundPad</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete all binds</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_1</tabstop>
  <tabstop>lineEdit_1</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>pushButton_13</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_12</tabstop>
  <tabstop>pushButton_11</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_10</tabstop>
 </tabstops>
 <resources>
  <include location="CMakeLists.qrc"/>
 </resources>
 <connections/>
</ui>
